doc/data-formats.md

```diff
@@ -91,26 +91,39 @@ See [the data description file](data-description.md) for more information about
 
 If your analysis requires de-duplicated gene symbols as row names, please use the collapsed matrices provided as part of the data download ([see below](#collapsed-expression-matrices)).
 
-#### Gene Fusion Data
+### Gene Fusion Data
 
 Gene Fusions produced by the [applied software packages](https://alexslemonade.github.io/OpenPBTA-manuscript/#rna-fusion-calling-and-prioritization) are provided as [Arriba TSV](doc/format/arriba-tsv-header.md) and [STARFusion TSV](doc/format/starfusion-tsv-header.md) respectively.
 These files are denoted with the prefix `pbta-fusion`.
 
 * `pbta-fusion-arriba.tsv.gz`
 * `pbta-fusion-starfusion.tsv.gz`
 
-#### Harmonized Clinical Data
+### Harmonized Clinical Data
 
 [Harmonized clinical data](https://alexslemonade.github.io/OpenPBTA-manuscript/#clinical-data-harmonization) are released as tab separated values in the following file:
 
 * `pbta-histologies.tsv`
 
-### Analysis Files
+#### Mapping Between DNA-seq and RNA-seq Data for the Same Sample
+
+Many analyses rely on examining DNA-seq (e.g., WGS/WXS/Panel) and RNA-seq data together.
+The identifiers in the `Kids_First_Biospecimen_ID` column of the `pbta-histologies.tsv` file will be non-overlapping for different experimental strategies, as these identifiers map to a _library or assay_.
+**Analysts should use the identifiers in the `sample_id` to connect DNA-seq and RNA-seq assays from the same sample.**
```

Comment:
I added some eye-drawing bold formatting and clarified where analysts can find relevant info about `sample_id` usage. Looks fine at this point, if you agree with my assessment of where analysts can find relevant code.